frontend/src/pages/personal/Personal.js

```diff
@@ -61,9 +62,10 @@ function Personal() {
       </div>
     );
   } else if (result && typeof result === 'object') {
+    const name = people.find(p => p.id === parseInt(person)).name;
     content = (
       <HeatMap
-        title={lastWords ? `Words: ${lastWords}` : 'Messages'}
+        title={lastWords ? `All messages for ${name} with words: ${lastWords}` : `All messages for ${name}`}
```

Comment:
Dit is incorrect, indien er een woord wordt meegegeven dan retourneerd hij altijd het aantal keren dat dat woord gebruikt is. Niet het aantal messages waarin dat woord gebruikt is